src/MonadTask.ts

```diff
@@ -48,3 +48,17 @@ export interface MonadTask3<M extends URIS3> extends MonadIO3<M> {
 export interface MonadTask3C<M extends URIS3, E> extends MonadIO3C<M, E> {
   readonly fromTask: <R, A>(fa: Task<A>) => Kind3<M, R, E, A>
 }
+
+/**
+ * @since 2.4.4
+ */
+export interface MonadTask4<M extends URIS4> extends MonadIO4<M> {
+  readonly fromTask: <S, R, E, A>(fa: Task<A>) => Kind4<M, S, R, E, A>
+}
+
+/**
+ * @since 2.4.4
+ */
+export interface MonadTask4C<M extends URIS4, E> extends MonadIO4C<M, E> {
```

Comment:
`MonadTask4` is required for `StateReaderTaskEither` but what's the use case for `MonadTask4C`?

Reply:
Currently any use case. Can only image if someone wants to implement `getStateReaderTaskValidation` but for this we would need a lot more than this interface. I will remove it for now cause it was more of a copy paste move.